tests/drivers/build_all/sensor/spi.dtsi

```diff
@@ -318,8 +318,20 @@ test_spi_icm42670: icm42670@3c {
 	gyro-fs = <2000>;
 };
 
-test_spi_bme680: bme680@3c {
+test_spi_bme680: bme680@27 {
 	compatible = "bosch,bme680";
-	reg = <0x3c>;
+	reg = <0x27>;
 	spi-max-frequency = <0>;
 };
+
+/* Requires the SPI to be atml,sam-spi
```

Comment:
I assume you meant to submit this as a separate commit?

Reply:
I left it in there so Tom & Maureen could see what I tried. It's been split up into a separate commit now that it works.